docs/maps/maps-getting-started.asciidoc

```diff
@@ -65,8 +66,8 @@ and lighter shades symbolize countries with less traffic.
 
 You must add the web log traffic property to the world countries so
 that the property is available for styling.
-You'll create a <<terms-join, terms join>> to link the vector source *World Countries* to 
-the {es} index `kibana_sample_data_logs` on the shared key iso2 = geo.src. 
+You'll create a <<terms-join, terms join>> to link the vector source *World Countries* to
+the {es} index `kibana_sample_data_logs` on the shared key iso2 = geo.src.
```

Comment:
Let's add some wording to better explain the terms join.  See the wording in the video script.